app/lib/pre_assembly/batch.rb

```diff
@@ -113,12 +123,6 @@ def discover_items_via_crawl(root)
       Dir.glob("#{root}/**/*")
     end
 
-    # A convenience method to return all ObjectFiles for all digital objects.
-    # Also used for stubbing during testing.
-    def all_object_files
-      digital_objects.map(&:object_files).flatten
-    end
```

Comment:
this was only used by the tests, in a somewhat artificial way, and the refactoring to use `Enumerable` more consistently lead to this method's use being more a hindrance than a help.